package.json

```diff
@@ -15,18 +15,26 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:each"
+    "test": "ember try:each && mocha",
```

Comment:
Can we split this into separate test commands?

```json
"scripts": {
  "test": "npm run test:browser && npm run test:node",
  "test:browser": "ember try:each",
  "test:node": "mocha"
}
```

Reply:
Great idea to separate them. The node test takes time, because if needs to install the fixture apps 👍